babel.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
- [ ] Kindly, make sure your app runs correctly on your machine. 👍🏻 
- 🤔 I have followed the steps provided in the README and I'm still getting the same error. Please solve this 😄 

![image](https://github.com/Saadat123456/hello-rails-react/assets/79058364/214cfa55-2f41-474c-bae3-09a736cde4c8)

Reply:
@BenMukebo mmmm yeah in my computer its working very well but i will check this<img width="1440" alt="Captura de pantalla 2023-05-09 a la(s) 13 50 11" src="https://github.com/PierreBarba/Ruby-on-Rails-React/assets/99061965/b6beb689-535d-4d04-b0fe-2068927e88ee">

<img width="1440" alt="Captura de pantalla 2023-05-09 a la(s) 19 37 04" src="https://github.com/PierreBarba/Ruby-on-Rails-React/assets/99061965/572f406a-2288-479c-a956-57b0467dbd68">

Reply:
- Great 👍🏻